src/Dapr.Client/DaprClient.cs

```diff
@@ -24,16 +24,54 @@ public abstract class DaprClient
         /// <param name="cancellationToken">A <see cref="CancellationToken" /> that can be used to cancel the operation.</param>
         /// <typeparam name="TData">The data type of the object that will be serialized.</typeparam>
         /// <returns>A <see cref="Task" /> that will complete when the operation has completed.</returns>
-        public abstract Task PublishEventAsync<TData>(string pubsubName, string topicName, TData data, CancellationToken cancellationToken = default);
+        public abstract Task PublishEventAsync<TData>(
```

Comment:
Suggestion - define this as non-abstract and have it call the longer overload here - that way there's fewer "implementation" methods

Reply:
nvm, I see that everything on this class is abstract for the most part. Seems best to follow the pattern.